template/typescript/src/cli.tsx

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env node
+import React from 'react';
+import {render} from 'ink';
+import meow from 'meow';
+import {App} from './ui';
+
+const cli = meow(`
+	Usage
+	  $ %NAME%
+
+	Options
+		--name  Your name
+
+	Examples
+	  $ %NAME% --name=Jane
+	  Hello, Jane
+`);
+
+render(React.createElement(App, cli.flags));
```

Comment:
Hey, thanks for this PR!! It helps me set up my ink project with Typescript. I'd love to see this PR to be merged 🙌 

Btw, in typescript, I have to do something like this to make it works 🤔 

```suggestion
render(React.createElement(App, cli.flags as never));
```